scenario/utils/relayOptimismMessage.ts

```diff
@@ -0,0 +1,81 @@
+import { DeploymentManager } from '../../plugins/deployment_manager';
+import { impersonateAddress } from '../../plugins/scenario/utils';
+import { setNextBaseFeeToZero, setNextBlockTimestamp } from './hreUtils';
+import { Event } from 'ethers';
+
+/*
+The Optimism relayer applies an offset to the message sender.
+
+applyL1ToL2Alias mimics the AddressAliasHelper.applyL1ToL2Alias fn that converts
+an L1 address to its offset, L2 equivalent.
+
+https://optimistic.etherscan.io/address/0x4200000000000000000000000000000000000007#code
+*/
+function applyL1ToL2Alias(address: string) {
+  const offset = BigInt('0x1111000000000000000000000000000000001111');
+  return `0x${(BigInt(address) + offset).toString(16)}`;
+}
+
+export default async function relayOptimismMessage(
```

Comment:
In order to test the Optimism proposal (that we will eventually write), we need to add token bridging to this function  like the Polygon relayer.